### ISSUE TITLE
loadRNASeq fails if sampleMetadataFile is passed as an argument

### ISSUE THREAD

**Issue by AngermannBR** [hbc/bcbioRNASeq#72]
Hi,

`loadRNASeq()` fails with the error 
```
Error in (function (classes, fdef, mtable)  : 
  unable to find an inherited method for function 'readSampleMetadataFile' for signature '"missing"'
```
this is due to `loadRNASeq()` calling `readSampleMetadataFile()` as shown below.
```
        sampleMetadata <- readSampleMetadataFile(
            file = sampleMetadataFile,
            lanes = lanes)
```
the file name is passed to the `file` argument. However, `readSampleMetadataFile()` expects the file name to be the `object` argument.

Best,
Bastian

**Comment by mjsteinbaugh** [comment]
Thanks I'll look into patching this with an update

**Comment by mjsteinbaugh** [comment]
@AngermannBR I just fixed this on the develop branch (v0.1.5):
https://github.com/hbc/bcbioRNASeq/blob/develop/R/loadRNASeq.R#L181

I'm finishing some additional documentation updates for this release and will merge onto the master branch soon.

Thanks!
Mike
